go.mod

```diff
@@ -8,6 +8,7 @@ require (
 	github.com/google/uuid v1.1.1
 	github.com/huandu/xstrings v1.2.0
 	github.com/imdario/mergo v0.3.7
+	github.com/shopspring/decimal v0.0.0-20190905144223-a36b5d85f337
```

Comment:
I was just looking at this package in depth and I have some concerns as it appears to be mostly unmaintained. For example:

* There are only two commits this calendar year
* Numerous bug fixes, improvements, and the move to modules are all sitting as PRs without responses
* There are bugs listed in the issue queue. I wonder if our users will run into these

The company that maintained this package was bought and the service that was using this appears to have been shut down. It looks to have issues while being unmaintained which gives me reason to pause.

I like what it does to make working with floats more practical.

Reply:
Do any of the bugs impact this usage?